docs/software/serialvtx.md

```diff
@@ -0,0 +1,87 @@
+---
+template: main.html
+description: Serial VTX Control
+---
+
+![Software Banner](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-Hardware/master/img/software.png)
+
+!!! note "NOTE"
+    SmartAudio/Tramp are loosely-defined protocols with edge cases, misinterpretations, and shoddy cloned implementations. ExpressLRS *does not* intend to support the vast majority of VTXes. SmartAudio was tested on a TBS Unify Pro32 HV, and Tramp was tested on an ImmersionRC Tramp HV, and both were tested with OpenVTX. It may work on other hardware, but it's not guaranteed and changes won't be made to accommodate broken implementations.
+
+## Description
+
+ExpressLRS now has support for controlling an external video transmitter through SmartAudio/Tramp protocols. This is a powerful feature for PWM+FPV users who don't have a flight controller to manage their VTX settings.
+
+## Hardware Requirements
+
+Using serial VTX control requires at least one UART TX pin assigned on your receiver. For most PWM RXes, you will need to assign a pin as Serial TX using the Lua script "Other devices" section, or using the [RX Web UI](/quick-start/webui/).
+
+!!! note "NOTE"
+    For _ESP8266/ESP8285_ (5ch/6ch/7ch PWM receivers) the Serial TX output **must be assigned to the physical PWM channel 2 pin**. For the ESP32 (BetaFPV SuperP), the Serial TX pin can be assigned to almost any available pin.
+
+## Software Requirements
+
+Ensure both your transmitter module and your receiver are up to date with the latest release versions; follow the [Firmware Update Guide](/quick-start/getting-started/) for detailed instructions.
+
+The minimum version to use this feature is:
+- Receiver firmware: `3.5.0`
+
+## Setting up serial VTX control (via Lua)
+
+1. Open the ExpressLRS LUA script, scroll to the bottom and select `Other Devices`, then select your receiver.
+
+1. Select `Output Map`, and assign an available `Output Ch` as either `Serial TX` or `Serial2 TX`.
+
+1. Select "BACK" to return to the main RX menu
+
+1. Select `Protocol` or `Protocol2` (depending on whether you chose `Serial TX` or `Serial2 TX` above) and set it to `SmartAudio` or `Tramp`. This configures the RX to output SmartAudio or Tramp on the previously-selected pin.
```

Comment:
Are SA/ Tramp supported on the main UART a.k.a `Protocol`?
I thought they were only available on serial2 a.k.a `Protocol2`?

I can't see them in this enum:
https://github.com/ExpressLRS/ExpressLRS/blob/bd5fcc7a0b23475981de83ebd6654d90f1a16b93/src/include/common.h#L216

Only in this one:
https://github.com/ExpressLRS/ExpressLRS/blob/bd5fcc7a0b23475981de83ebd6654d90f1a16b93/src/include/common.h#L229